tests/fixtures/joint.py

```diff
@@ -224,3 +228,57 @@ async def populated_fixture(
         example_file=EXAMPLE_FILE,
         joint_fixture=joint_fixture,
     )
+
+
+@dataclass
+class CleanupFixture:
+    """Fixture for cleanup test with DAO and test files"""
+
+    mongodb_dao: DrsObjectDaoPort
+    joint_fixture: JointFixture
+    cached_id: str
+    expired_id: str
+
+
+@pytest_asyncio.fixture
+async def cleanup_fixture(
+    joint_fixture: JointFixture,
+) -> AsyncGenerator[CleanupFixture, None]:
+    """Set up state for and populate CleanupFixture"""
+
+    # create AccessTimeDrsObjects for valid cached and expired cached file
```

Comment:
If we don't reuse this accross tests, maybe it is more transparent to directly put the setup logic into the test itself instead of a dedicated fixtures.

Reply:
I thought the idea was to move setup code out of the actual tests into fixtures?

Reply:
The way how the other fixtures are usually built in hexkit is that they expose methods for setting the state (e.g. s3_fixture has populate_file_objects method), however, typically they don't set the state themselves. Like that it is very easy to read which state a test is using.

Reply:
Still, there were a few PR's where I had the setup in the tests and was told it would be better to move that into a fixture.
As long as we don't use those downstream (i.e. as dependencies of other fixtures) this should be ok.
There's something in the refactoring notes about harmonizing fixture usage, so we can tackle the Do's and Dont's transparently when we take care of that.